src/components/PeoplePage.tsx

```diff
@@ -1,31 +1,83 @@
+import { useEffect, useMemo, useState } from 'react';
+import { useSearchParams } from 'react-router-dom';
 import { PeopleFilters } from './PeopleFilters';
 import { Loader } from './Loader';
 import { PeopleTable } from './PeopleTable';
+import { Person } from '../types/Person';
+import { getPeople } from '../api';
+import { filterTable } from '../utils/filterTable';
+// import { sortTable } from '../utils/sortTable';
 
 export const PeoplePage = () => {
+  const [peoples, setPeoples] = useState<Person[]>([]);
```

Comment:
```suggestion
  const [people, setPeople] = useState<Person[]>([]);
```
people are already plural